_sass/blog.scss

```diff
@@ -0,0 +1,39 @@
+.blog-container{
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    // margin-left: 10px;
```

Comment:
If you don't need this style I'd remove it. No need to push commented code that isn't doing anything 😉